src/page/search/advanced-search/example/index.jsx

```diff
@@ -0,0 +1,344 @@
+                        },
+                        {
+                            id: countId++,
+                            name: 'Essen'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Dortmund'
+                        }
+                    ],
+                    IDF: [
+                        {
+                            id: countId++,
+                            name: 'Paris'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Lyon'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Limeil'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Le Plessis-Robinson'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Bercy'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Marne la Vallée'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Antony'
+                        }
+                    ],
+                    Gironde: [
+                        {
+                            id: countId++,
+                            name: 'Bordeaux'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Mérignac'
+                        },
+                        {
+                            id: countId++,
+                            name: 'Langon'
+                        }
+                    ]
+                };
+
+                let list = [];
+
+                for(let i = 0; i< groups.France.length; i++)
+                    list.push(groups.France[i]);
+
+                for(let i = 0; i< groups.Germany.length; i++)
+                    list.push(groups.Germany[i]);
+
+                let name = '';
+                const showGroup = crit.data.facets.length > 0 ? true : false;
+                let group = [];
+
+                if(showGroup) {
+                    name = crit.data.facets[0].value;
+                    group = groups[name];
+                }
+                else
+                group = list;
+
+                const data = {
+                    facets: facets,
+                    'list': group,
+                    totalCount: group.length
+                };
+                success(data);
+            }, 300);
+        });
+    }
+};
+
+const service = {
+    unscoped: getSearchService(false),
+    scoped: getSearchService(true)
+};
+
+const Line = React.createClass({
+    mixins: [FocusComponents.list.selection.line.mixin],
```

Comment:
Outside the React.createClass `const linePreset = FocusComponents.list.selection.line.mixin]`